compiler/luci/pass/src/ResolveFormerCustomOpPass.cpp

```diff
@@ -47,15 +55,6 @@ bool resolve_with_BroadcastTo(luci::CircleCustom *node)
   broadcastTo->name(name);
   luci::add_origin(broadcastTo, luci::get_origin(node));
 
-  auto customOut = loco::succs(node);
-
-  assert(customOut.size() == 1);
-
-  // check if the data type of output is same with the one of the input feature map.
-  auto output = loco::must_cast<luci::CircleNode *>(*customOut.begin());
-  if (input->dtype() != output->dtype())
-    return false;
```

Comment:
when return false is called, `broadcastTo` objected is useless.
this is not critical as not used nodes will be cleaned later.